com.sublimetext.three.json

```diff
@@ -0,0 +1,193 @@
+{
+    "app-id": "com.sublimetext.three",
+    "runtime": "org.gnome.Sdk",
```

Comment:
What does this use from gnome not in fdo?

Reply:
GTK 2.

Reply:
Might be more size efficient to use the electron baseapp and fdo.

Reply:
GNOME SDK runtime:
  - app: 52.0 MB
  - runtime: 2.1 GB

Freedesktop SDK runtime / Electron base app:
  - app: 93.9 MB
  - runtime: 1.9 GB

I guess the more important question is which SDK is your average downloader likely to have installed?

Reply:
Your average user probably has no Sdk beforehand. Since Electron apps are some of the more popular ones on Flathub they probably already have those bits too.

Either choice isn't the end of the world but thats what I'd do.

Reply:
Yeah, makes sense to keep it this way. The only counter I can think of is that if they have org.gnome.Builder, they'll have org.gnome.Sdk already. But maybe someone using Builder is less likely to want this?

Reply:
Atom, Emacs, VSCode, etc are far more popular than Builder but you are also right people probably don't have multiple editors.